tests/cpp/ml_demo/src/eeg_metrics.cpp

```diff
@@ -11,6 +11,7 @@
 #include "board_shim.h"
 #include "data_filter.h"
 #include "ml_model.h"
+#include  <vector>
```

Comment:
can we move SVM to another test\file? This example is used in docs on public page as code sample. I want to keep it short and simple for demo purposes